docs/features/configuring_workflows.md

```diff
@@ -0,0 +1,21 @@
+# Configuring Your Workflows
+
+DAG Factory allows you to define workflows in a structured, configuration-driven way using YAML files.
+
+## Key Elements of Workflow Configuration
+
+- **dag_id**: Unique identifier for your DAG.
+- **default_args**: Common arguments for all tasks.
+- **schedule**/**schedule_interval**: Specifies the execution schedule.
+- **tasks**: Defines the [Airflow tasks](https://airflow.apache.org/docs/apache-airflow/stable/core-concepts/tasks.html) in your workflow.
+
+### Example DAG Configuration
+
+```title="example_dag_factory.yml"
+--8<-- "dev/dags/example_dag_factory.yml:example_dag_yaml_configuration"
+```
+
+### Check out more configuration params
+
+- [Environment variables](./environment_variables.md)
```

Comment:
@pankajkoti what do you think if we grouped configuration-related docs inside `docs/configuration`?

Reply:
done so, thanks.